wrapper/src/test/java/integration/container/tests/CustomEndpointTest.java

```diff
@@ -80,139 +78,123 @@
 @Order(16)
 public class CustomEndpointTest {
   private static final Logger LOGGER = Logger.getLogger(CustomEndpointTest.class.getName());
-  protected static final String oneInstanceEndpointId = "test-endpoint-1-" + UUID.randomUUID();
-  protected static final String twoInstanceEndpointId = "test-endpoint-2-" + UUID.randomUUID();
-  protected static final Map<String, DBClusterEndpoint> endpoints = new HashMap<String, DBClusterEndpoint>() {{
-      put(oneInstanceEndpointId, null);
-      put(twoInstanceEndpointId, null);
-    }};
+  protected static final String endpointId = "test-endpoint-1-" + UUID.randomUUID();
+  protected static DBClusterEndpoint endpointInfo;
 
   protected static final AuroraTestUtility auroraUtil = AuroraTestUtility.getUtility();
-  protected static final boolean reuseExistingEndpoints = false;
+  protected static final boolean reuseExistingEndpoint = false;
 
   protected String currentWriter;
 
   @BeforeAll
-  public static void createEndpoints() {
+  public static void createEndpoint() {
     TestEnvironmentInfo envInfo = TestEnvironment.getCurrent().getInfo();
     String clusterId = envInfo.getAuroraClusterName();
     String region = envInfo.getRegion();
 
     try (RdsClient client = RdsClient.builder().region(Region.of(region)).build()) {
-      if (reuseExistingEndpoints) {
-        waitUntilEndpointsAvailable(client, clusterId);
+      if (reuseExistingEndpoint) {
+        waitUntilEndpointAvailable(client);
         return;
       }
 
       List<TestInstanceInfo> instances = envInfo.getDatabaseInfo().getInstances();
-      createEndpoint(client, clusterId, oneInstanceEndpointId, instances.subList(0, 1));
-      createEndpoint(client, clusterId, twoInstanceEndpointId, instances.subList(0, 2));
-      waitUntilEndpointsAvailable(client, clusterId);
+      createEndpoint(client, clusterId, instances.subList(0, 1));
+      waitUntilEndpointAvailable(client);
     }
   }
 
-  private static void deleteEndpoints(RdsClient client) {
-    for (String endpointId : endpoints.keySet()) {
-      try {
-        client.deleteDBClusterEndpoint((builder) -> builder.dbClusterEndpointIdentifier(endpointId));
-      } catch (DbClusterEndpointNotFoundException e) {
-        // Custom endpoint already does not exist - do nothing.
-      }
-    }
-
-    waitUntilEndpointsDeleted(client);
-  }
-
-  private static void waitUntilEndpointsDeleted(RdsClient client) {
```

Comment:
This used to be called after deleting the test custom endpoint after all tests in the file had been completed. I removed it to save some time. Each custom endpoint will have a unique ID, so even though other tests could start running while the custom endpoint is still being deleted, no other tests should be require it to be fully deleted before running. Let me know if you think I should add it back